### PR TITLE
[#1825] Give more info on the File Size Limit in UG

### LINKED ISSUE #1825: User Guide: Give more info on the File Size Limit
In https://reposense.org/ug/configFiles.html#repo-config-csv the following description of the `File Size Limit` appears.

image

Suggestions/questions:
* Specify the default file size referred in the description
* Give an example
* Also mention how the file size limit (default or otherwise) affects the report?
* Why is this column marked as a multi-valued column?

### PR DESCRIPTION
**Pull request by gok99**
Fixes #1825

## Proposed commit message

```
The documentation on File Size Limit lacks examples and information on
behavior and usage.

Let's improve the documentation by providing more details about the
feature functionality.
```

### CODE REVIEW

**Review by Zhou-Jiahao-1998 — APPROVED** [review]
LGTM

File: docs/ug/configFiles.md [hunk 1]
```diff
@@ -40,9 +40,9 @@ Given below are the details of the various config files used by RepoSense.
...
+| File Size Limit<sup>+</sup> | Enter a file size limit for the repository in bytes as a single number without units (for a size limit of 1MB for example, enter 1000000). This file size limit will override the default file size limit (500KB). If file analysis is not skipped, files exceeding the file size limit will be marked as ignored and only the file name and line count will be reflected in the report. |
 | Ignore File Size Limit | Enter **`yes`** to ignore both the default file size limit and the file size limit possibly set by the user in `repo-config.csv`. |
-| Skip Ignored File Analysis | Enter **`yes`** to ignore analysis of files exceeding the file size limit entirely. By default, files exceeding the size limit are analyzed and line count information (but not line diffs) for these files are included in the report. This option can significantly improve report generation time. |
+| Skip Ignored File Analysis | Enter **`yes`** to ignore analysis of files exceeding the file size limit entirely. If file analysis is ignored, all information about the file will be omitted from the generated report. This option can significantly improve report generation time. |
```
Comment (damithc) [inline docs/ug/configFiles.md:45]: `If file analysis is ignored`: did you mean `If file analysis is skipped`?

File: docs/ug/configFiles.md [hunk 2]
```diff
@@ -40,9 +40,9 @@ Given below are the details of the various config files used by RepoSense.
...
 | Ignore Authors List<sup>*+</sup> | The list of authors to ignore during analysis. Authors should be specified by their [Git Author Name](#a-note-about-git-author-name). |
 | Shallow Cloning | Enter **`yes`** to clone the repository using Git's shallow cloning functionality. This option can significantly reduce the time taken to clone large repositories. However, the option should ideally be disabled for smaller repositories where the `.git` file is smaller than 500 MB, as it would create overhead. |
-| File Size Limit<sup>+</sup>  | Enter a file size limit for the repository in bytes as a single number without units, that will override the default file size limit. |
+| File Size Limit<sup>+</sup> | Enter a file size limit for the repository in bytes as a single number without units (for a size limit of 1MB for example, enter 1000000). This file size limit will override the default file size limit (500KB). If file analysis is not skipped, files exceeding the file size limit will be marked as ignored and only the file name and line count will be reflected in the report. |
```
Comment (damithc) [inline docs/ug/configFiles.md:43]: Perhaps omit `If file analysis is not skipped,`? The skipping is explained further down. Mentioning it here (while it makes the explanation more accurate), might puzzle the reader.

**Review by damithc — COMMENTED** [review]
Just a couple of cosmetic suggestions.

**Pull request merged**